example-conf.yml

```diff
@@ -566,6 +569,25 @@ relay:
     #
     #clientbot_modesync: none
 
+###
+#
+# Serv Protect
+#
+# If you use this please uncomment the following values
+# and set them to what you want them to be.
+#
+###
+
+#servprotect:
+    # THINK: LENGTH KILL/SAVE's in TIME seconds
```

Comment:
This line is ambiguous, just write something like "This block configures the servprotect plugin. You don't need this if you aren't using it."